### ISSUE TITLE
[0.9.2.4 beta Release-156]Rent crash NRE due to client cache of old invite to non existing property.

### ISSUE THREAD

**Issue by IcedForge** [StrangeLoopGames/EcoIssues#20936]
Reproduction
User 1 Rents plot
User 1 leaves plot
User 2 (Landlord) changes rent move in fee and reinvites User 1
User 1 accepts new invite which results in server crash with following dump

```
--BEGIN DUMP--
Dump Time
03/14/2021 13:49:12

Exception
Exception: NullReferenceException
Message:Object reference not set to an instance of an object.
Source:Eco.Gameplay

System.NullReferenceException: Object reference not set to an instance of an object.
   at Eco.Gameplay.Property.RentForProperty.ValidateAndGetFeeInfo(User user)
   at Eco.Gameplay.Property.Residency.BecomeResident(User user)
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_1(Object state)
   at System.Threading.QueueUserWorkItemCallbackDefaultContext.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()

--END DUMP--
```
Need to check this in 9.3 if this occurs ( since we have changes to invite consumption now )

**Comment by IcedForge** [comment]
I dont think this warrants a hotfix as 9.2 has rent as experimental even though its annoying if this does not occur on 9.3 as we redid the invite stuff.

**Comment by WeaselDog** [comment]
@IcedForge Is it fixed in 9.3 currently?

**Comment by IcedForge** [comment]
Seems to function properly for 9.3 with the reworked invite system ( also tried using the reproduction steps that exploded on 9.2 with demographics etc and it worked without problems )

**Comment by WeaselDog** [comment]
Closing as fixed for 9.3
